form-test-tom/src/App.jsx

```diff
@@ -0,0 +1,105 @@
+import {useForm} from './useForm'
+import React from 'react'
+import params from './formParams.js'
+
+const animaux = ['bear','tiger','snake','donkey']
```

Comment:
hoist this variable out of scope. It doesn't change, so it doesn't need wiping and re-instating into memory every time the component loads